### ISSUE TITLE
Ghast fireballs always deflect

### ISSUE THREAD

**Issue by DarkBladeNemo** [STS15/Fargos-Talismans#39]
### Modpack

No

### Neoforge Version

21.1.90

### Mod Version

1.21-2.2.21

### Describe the issue

Ghast fireballs always reflect even when you are completely naked. Deflection seems to happen regardless of forward facing direction.

Managed to reproduce with only Fargo's Talismans, Curios API Continuation, EventsLib and Patchouli installed.

### Expected behaviour

For fireballs not to deflect automagically

**Comment by STS15** [comment]
You're right, this was a mistake on my part.  I've added a fix for this in dev.  Will be fixed on v2.2.3.  Thanks for testing and letting me know!
